charts/netdata/Chart.yaml

```diff
@@ -1,28 +1,28 @@
 apiVersion: v2
 name: netdata
-version: 3.6.2
+version: 3.6.3
 description: Real-time performance monitoring, done right!
 type: application
 keywords:
-  - alerting
-  - metric
-  - monitoring
+- alerting
+- metric
+- monitoring
 home: https://netdata.cloud/
 sources:
-  - https://github.com/netdata/helmchart
-  - https://github.com/netdata/netdata
+- https://github.com/netdata/helmchart
+- https://github.com/netdata/netdata
 maintainers:
-  - name: Ilya Mashchenko
-    email: ilya@netdata.cloud
-  - name: Konstantinos Natsakis
-    email: knatsakis@netdata.cloud
-  - name: Mansour Behabadi
-    email: mansour@netdata.cloud
-  - name: Vladimir Ryumin
-    email: vryumin@gmail.com
-  - name: Karuppiah Natarajan
-    email: karuppiah7890@gmail.com
-  - name: Oleksii Kravchenko
-    email: kamikaze.ua@gmail.com
+- name: Ilya Mashchenko
```

Comment:
why not indent sequence value?

Reply:
I was using the script from #194 and forgot to fix the indentation.

I still need to test if the script works as we need it to with ruamel.